crypto/crypto/src/ed25519.rs

```diff
@@ -42,6 +42,8 @@ use mirai_annotations::*;
 use serde::Serialize;
 use std::{cmp::Ordering, fmt};
 
+pub use ed25519_dalek;
```

Comment:
So the biggest issue is that the Ed25519 keys exported form the crytpo crate do not support signing of arbitrary messages, only BCS serialized messages. So in order to actually test dual-attestation we need to have access to ed25519 keys that can actually sign arbitrary messages.

Reply:
btw there is signing of arbitrary messages, but it's a private function https://github.com/diem/diem/blob/main/crypto/crypto/src/ed25519.rs#L107

Reply:
@kchalkias Yeah I'm aware of that function, and that it is private. I figured instead of making it public i would instead just export the underlying ed25519 library and use those keys. This way we can't misuse the keys defined in the crypto crate.

Reply:
yeah it completely makes sense.